Makefile

```diff
@@ -9,7 +9,7 @@ CLANG_FORMAT ?= clang-format-10
 PIP ?= python -m pip
 
 # versions used in CI
-PYTORCH_VERSION ?= dev20220820
+PYTORCH_VERSION ?= b82c74da07430ba4a221403d383eeb27de04f7f7
```

Comment:
This needs to be the name of a nightly build, the list can be found here:
https://download.pytorch.org/whl/nightly/torch/